zenodo/modules/records/serializers/dcat.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Zenodo.
+# Copyright (C) 2016 CERN.
+#
+# Zenodo is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Zenodo is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Zenodo; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Datacite to DCAT serializer."""
+
+from __future__ import absolute_import, print_function
+
+import os
+from werkzeug.utils import cached_property
+
+from invenio_oaiserver.response import getrecord
+from lxml import etree as ET
+
+
+class DCATSerializer(object):
+    """DCAT serializer for records"""
+
+    @cached_property
+    def xslt_transformation(self):
+        current_location = os.path.abspath(os.path.dirname(__file__))
+        file_path = os.path.join(
+            current_location, "../data/datacite-to-dcat-ap.xsl")
+        with open(file_path, 'r') as f:
```

Comment:
To avoid using relative paths, you can use the `pkg_resources.resource_stream('zenodo.modules.records', 'data/datacite-to-dcat-ap.xsl')` instead